src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>${groupId}</groupId>
+  <artifactId>${artifactId}</artifactId>
+  <packaging>war</packaging>
```

Comment:
what do you think about forcing the default project to be multi-module? we don't want to imply that you can put servlets etc in here, we want to encourage client vs server (and possibly shared)

Reply:
i think, it's ok for now to do a single project app, but it's only for now. From my point of view, it ll be great to have ability to generate multi-module project with gwt-rpc on it.

Reply:
As long as we emphasize that no server code can go in this - it is only a `.war` file since it is a zip, and should contain no java that can't run in the browser

Reply:
hmm, i think we should keep war as a default packaging type, coz it's the expected behavior for out users

Reply:
Yeah a war as the output definitely makes sense (gwt-app is actually a war), but to merge in server classes, you need to overlay them or something.